src/CommandingOfficers/Commander.java

```diff
@@ -251,22 +251,27 @@ public void activateAbility(CommanderAbility ability, MapMaster map)
     myActiveAbility = ability;
   }
 
-  public void modifyAbilityPower(double amount)
+  public void modifyAbilityPower(int amount)
   {
     myAbilityPower += amount;
     if( myAbilityPower < 0 )
       myAbilityPower = 0;
-    double maxPower = getMaxAbilityPower();
+    int maxPower = getMaxAbilityPower();
     if( myAbilityPower > maxPower )
       myAbilityPower = maxPower;
   }
+  public void modifyAbilityStars(int amount)
+  {
+    // TODO: Make this scale with fatigue
+    modifyAbilityPower(amount * CHARGERATIO_FUNDS);
```

Comment:
Is.. fatigue a thing in AW? I'm admittedly less familiar with DoR ability mechanics, but I don't remember that.

Reply:
Uhhh, people started calling it "power fatigue" recently and it got stuck in my brain.
It's the thing where power costs scale up by 0.2x of the base cost each use for 9 or 10 uses.
DoR actually doesn't have it, since you can only charge in your zone (which shrinks after COP anyway).

Reply:
Ahh, gotcha. Just haven't heard that term before.